src/plots/plots.js

```diff
@@ -19,8 +19,8 @@ var axisIDs = require('../plots/cartesian/axis_ids');
 var Lib = require('../lib');
 var _ = Lib._;
 var Color = require('../components/color');
+var Grid = require('../components/grid');
```

Comment:
Do you want to go the last inch here and turn `Grid.method` into `Registry.getComponentMethod`? Then `grid` becomes an optional component, right?

Reply:
done in -> https://github.com/plotly/plotly.js/pull/2429/commits/9246fdad6a85f812d23900e772ae210bac52d283